pkg/controller/controllercmd/cmd.go

```diff
@@ -133,7 +133,7 @@ func hasServiceServingCerts(certDir string) bool {
 // you do not need to customize the controller builder. This method modifies config with self-signed default cert locations if
 // necessary.
 func (c *ControllerCommandConfig) AddDefaultRotationToConfig(config *operatorv1alpha1.GenericOperatorConfig, configContent []byte) (map[string][]byte, []string, error) {
-	certDir := "/var/run/secrets/serving-cert"
+	certDir := "/var/run/secrets/service-network-serving-certkey"
```

Comment:
This change looks wrong.  This is specific to use by the kube-apiserver (which doesn't use this library) and this is pointing to a file which is automatically reloaded.  What exactly are you fixing, for which binary, and can you provide a pod spec for what's failing?